lib/tasks/questionnaires.rake

```diff
@@ -4,7 +4,7 @@ namespace :questionnaires do
     # Due to mismatch in Gregorian<>Ethiopian calendars,
     # Dynamic forms initialization had a delay of 20+ days in Ethiopia.
     # To avoid this delay, we initialize dynamic forms 1 month in advance in Ethiopia.
```

Comment:
^ This comment needs to be updated to reflect the current situation.

From what I understand the main issue is:
* The Ethiopian calendar is broken up differently than the Gregorian calendar and thus out of sync, thus we need to initialise responses in advance.
* One month in advance is not sufficient, because Ethiopian months (ignoring Pagume) are 30 days, and Gregorian Months are usually 30 or 31 days, which creates a few days' delay between when the users expect the new form to initialise and when they actually initialise.
* Thus, 2 months in advance safely covers up for all potential delays.

Is this right?

Reply:
> One month in advance is not sufficient, because Ethiopian months (ignoring Pagume) are 30 days, and Gregorian Months are usually 30 or 31 days, which creates a few days' delay between when the users expect the new form to initialise and when they actually initialise.

Not exactly. The Ethiopian Calendar has 12 months of 30 days each, plus five or six additional days (sometimes known as the 13th month). More on ET time [here](https://ethiopianembassy.org/ethiopian-time/). 
Their reporting period for a month starts in a month and ends in another month. Whereas in Gregorian calendar, reporting period is starts and end in the same month. 
The ET reporting period for the 8th month is `7/21/2016 - 8/20/2016`.  In ET we initialise current month form at the start of Gregorian calendar which is  `8-23-2016`.  We will be initialising 8th month form on `8-23-2016` which is delayed by week. We thought we are be pre-initialising the forms in ET but in reality forms are created after the month with a weeks delay.